README.md

```diff
@@ -1,4 +1,5 @@
 # datamegh
+[![Build Status](https://travis-ci.org/leapfrogtechnology/datamegh.svg?branch=master)](https://travis-ci.org/leapfrogtechnology/datamegh)  
```

Comment:
Can we have a flat style badge? Like these -> https://github.com/leapfrogtechnology/sync-db
![image](https://user-images.githubusercontent.com/3315763/74591568-0a0de300-5041-11ea-9c9e-9f9a41dedf58.png)